_posts/wiki/2019-09-03-Undergrad-Guide.md

```diff
@@ -0,0 +1,158 @@
+---
+title: Getting Started as an Undergraduate in the HIRO Group
+description: Answering the questions you didn't know to ask
+permalink: join_us.html
```

Comment:
you should change the permalink to `undergrad_guide.html`
Also, you should add yourself as an author!